Assets/Editor/ModelInspector.cs

```diff
@@ -510,30 +684,41 @@ public override void OnPreviewGUI(Rect r, GUIStyle background)
                         "Mesh preview requires\nrender texture support");
                 return;
             }
-
+            
             Init();
 
             Assembly editorAssembly = Assembly.GetAssembly(typeof(EditorGUI));
             Type guiPreview = editorAssembly.GetType("PreviewGUI");
             MethodInfo drag2D = guiPreview.GetMethod("Drag2D");
-
-            if(m_Settings.displayMode != DisplayMode.UVLayout)
-                m_Settings.previewDir = (Vector2)drag2D?.Invoke(null, new object[] {m_Settings.previewDir, r});
-            //previewDir = PreviewGUI.Drag2D(previewDir, r);
             
-            if (Event.current.type == EventType.ScrollWheel && m_Settings.displayMode == DisplayMode.UVLayout)
+            if (Event.current.commandName == "FrameSelected") // compare against EventCommandNames.FrameSelected when on trunk
```

Comment:
Minor: in this function there's a whole bunch of `Event.current` calls overall, doing `var evt = Event.current` somewhere would be less typing.